content/en/docs/setup/install/helm/index.md

```diff
@@ -56,6 +56,11 @@ Follow this guide to install and configure an Istio mesh using
     See [Controlling the injection policy](/docs/setup/additional-setup/sidecar-injection/#controlling-the-injection-policy) for more info.
     {{< /warning >}}
 
+1. (Optional) Advanced Helm chart Customization
```

Comment:
I'm not sure this makes sense to add as a numbered item. It might be better to have this as a tip at the end of the section.